cernopendata/modules/fixtures/data/articles/cms-summer-student-report-2017/cms-summer-student-report-2017.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "title": "Improving educational content with high school teachers: A field report from our summer students",
+    "type": {"primary": "News"},
+    "body": {
+      "content": "cms-summer-student-report-2017.md",
+      "format": "md"
+    },
+    "experiment": "CMS",
+    "collections": [
+      {"experiment": "CMS"},
+      {"primary": "News" }
+    ],
+    "author": "Henna Silvennoinen, Mira Tengvall, Edith Villegas Garcia",
```

Comment:
I think this should be rather turned into a list  (`authors`) similarly as we do in records... and we shouldn't store `"author": "ALICE Collaboration"` in other news records ditto... But this would require schema changes etc. I guess we can take it afterwards? CC @ArtemisLav

Reply:
I agree and, if possible, let's apply this already for these records and we can amend the schema afterwards.

Reply:
Ok! :+1: